src/assets/js/pages/DashboardPage/index.js

```diff
@@ -45,6 +46,12 @@ class DashboardPage extends React.Component {
           {getRole(user.role) === getRole(Roles.ROLE_SPONSOR) &&
             <SponsorDashboard events={Object.values(events)} />}
         </div>
+        {getRole(user.role) >= getRole(Roles.ROLE_ADMIN) &&
+        <h2 className="event dashboard-page__btn">
```

Comment:
What is the .event class?
Also perhaps it'd be better to have this wrapped in a div that is defined to sit in the corner of the page, then add the button to that instead?
Can we also give it a better name than just "__btn", because that doesn't describe that it's in the corner specifically